### PR TITLE
create new user

### PR DESCRIPTION
**Pull request by at-hienpham**
## Description
Create new user
## Reference Requirements
- Task ID: TI-1

## Reference Isuees
#{issue_id}

## Migrations
YES | NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: app/Http/Controllers/Admin/UserController.php [hunk 1]
```diff
@@ -18,4 +19,38 @@ public function index()
...
+            'username' => $request->username,
+            'full_name' => $request->full_name,
+            'birthday' => $request->birthday,
+            'gender' => $request->gender,
+            'phone' => $request->phone,
+            'email' => $request->email,
+            'password' => $request->password,
+            'role_id' => $request->role_id,
+            'is_active' => $request->is_active,
+        ];
```
Comment (at-quangtran) [inline app/Http/Controllers/Admin/UserController.php]: nếu đã qua CreateUserRequest validate rồi thì ở đây chỉ cần  $userData = $request->all();

File: resources/views/admin/pages/users/index.blade.php [hunk 2]
```diff
@@ -6,11 +6,20 @@
...
+              </button>
+              {{session('message')}}
+            </div>
+          @endif
         <div class="header">
             <h2>{{__('user.admin.show.form_title')}}</h2>
             <a href="{{ route('admin.users.create') }}"
               class="btn bg-green waves-effect" style="margin-top: 30px;"> <i
-              class="material-icons">add</i> <span>{{ __('user.admin.show.create_user') }}</span>
+              class="material-icons">person_add</i> <span>{{ __('user.admin.show.create_user') }}</span>
```
Comment (at-quangtran) [inline resources/views/admin/pages/users/index.blade.php]: hardcode "person_add"

File: tests/Browser/Admin/Users/CreateUserTest.php [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+     *
+     * @return void
+     */
+    public function test_validate($name, $content, $message)
+    {
+        $this->browse(function (Browser $browser) use ($name, $content, $message) {
+            $browser->loginAs($this->user)
+                    ->visit(new CreateUser)
+                    ->type($name, $content)
+                    ->press('Create User');
```
Comment (at-quangtran) [inline tests/Browser/Admin/Users/CreateUserTest.php]: thiếu ->assertSee($message);

Comment (at-hienpham) [reply tests/Browser/Admin/Users/CreateUserTest.php]: ->visit(new CreateUser) dòng này nó tự động gọi qua file CreateUser.php để thực hiện ->assertSee($message); rồi anh

**Pull request merged**